.github/workflows/fetch_stats.yml

```diff
@@ -9,17 +9,39 @@ on:
     types:
       - created
       # Allow manual triggering of workflow by adding a label
+  workflow_dispatch:
 
 jobs:
-  build:
+  fetch:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
`fail-fast: false` means even if one or more scrapers in the matrix errors, the rest will keep running.